docs/release-notes/2.12.0.md

```diff
@@ -72,3 +72,16 @@ performance.
 * [PMM-6632](https://jira.percona.com/browse/PMM-6632): Node Summary - Virtual Memory Utilization chart: incorrect formulas
 * [PMM-6537](https://jira.percona.com/browse/PMM-6537): MySQL InnoDB Details - Logging - Group Commit Batch Size: giving incorrect description
 * [PMM-6055](https://jira.percona.com/browse/PMM-6055): PMM Inventory - Services: 'Service Type' column empty when it should be 'External' for external services
+
+
+## Known Issues
+
+* [PMM-7092](https://jira.percona.com/browse/PMM-7092): Update docker pmm-server 2.11.1 to 2.12.0 results in an unhealthy container.
+
+Workaround:
+It's needed to be created a folder for one of the components. Before starting pmm-server 2.12.0 execute:  

```

Comment:
"A folder is not created on container upgrade and will need to be created for one of the components.  Before starting..."